src/Controller/CaseController.php

```diff
@@ -779,4 +779,24 @@ public function applyIdentifierData(IdentificationHelper $identificationHelper,
 
         return $identificationHelper->fetchIdentifierData($identifier, $type);
     }
+
+    /**
+     * @Route("/get/inspection-address", name="case_get_inspection_address", methods={"GET", "POST"})
+     */
+    public function getInspectionAddress(CaseEntityRepository $repository, AddressHelper $addressHelper, Request $request): Response
+    {
+        $identifier = $request->request->get('identifier');
+
+        if (empty($identifier)) {
+            return new JsonResponse(['address' => '']);
+        }
```

Comment:
I'm not sure this micro optimization is worthwhile. I would just pass (the possible empty) identifier to `findOneBy` and let the check for no case return the empty address.

Reply:
Sure, removed.